CHANGELOG.md

```diff
@@ -10,6 +10,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - `CONJUR_LOG_LEVEL` for the Conjur container can now be configured by setting the
   `logLevel` value, or updated using `helm upgrade` [cyberark/conjur-oss-helm-chart#77](https://github.com/cyberark/conjur-oss-helm-chart/issues/77)
 
+### Changed
+- `account` now accepts two values, `account.create`, a boolean, and `account.name`, a string. 
+  These values allow you to configure the creation of a Conjur account on container startup, and 
```

Comment:
Should we note the default values for these options? Probably makes it clearer.

Reply:
is this a breaking change for current consumers of the helm chart?

Reply:
Yes, this will be breaking. Maybe for the short term we could make this non-breaking by keeping the existing `account` chart value and add an `accountCreate`, with `accountCreate` defaulting to `false` (so that current consumers' tooling doesn't break). And then for next major release, use the hierarchy that Bradley proposes here?

Reply:
is it possible to support `account` having a value (that internally gets mapped to `account.name`) so that we can document `account.create` and `account.name`, but still support (undocumented) `account`?

if that is not possible (with a test, since we don't want to inadvertently drop that support), then `accountCreate` sounds fine.

Reply:
We can assign [multiple types](https://cswr.github.io/JsonSchema/spec/multiple_types/) to `account`, so it can be an object or a string. I'll adjust things, but that should preserve backwards compatibility.